Makefile

```diff
@@ -32,12 +32,17 @@ docclean:
 livehtml:
 	$(MAKE) -C doc $@
 
+# The first dist target here exports a variable to the environment that keeps tar
```

Comment:
What does it mean to have the same target twice? Is that well-formed and/or maybe a GNUism?

Reply:
Yah, looks like `export` might be a GNUism. That fails on FreeBSD. I guess I could just move setting the variable down before the two tar statements.

Reply:
I'd say pull it down before the tars, then at least if clear what's happening.